### PR TITLE
chore: remove unused imports in examples

### PR DESCRIPTION
**Pull request by ydcjeff**
Description: remove unused imports in examples
To check this PR
- check out this branch
- remove F401 in setup.cfg
- run `flake8 examples`

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: examples/references/classification/imagenet/code/utils/exp_tracking.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from ignite.contrib.engines import common
 
 try:
-    import polyaxon_client.tracking
+    import polyaxon_client.tracking  # noqa: F401
```
Comment (vfdev-5) [inline examples/references/classification/imagenet/code/utils/exp_tracking.py:13]: As we do not modify our own setup.cfg, technically, there is no need to add `# noqa: F401`, right ?

**Comment by vfdev-5** [comment]
Thanks for the PR @ydcjeff ! This is helpful !

File: examples/references/classification/imagenet/code/utils/exp_tracking.py (hunk 1 above)
Comment (ydcjeff) [reply examples/references/classification/imagenet/code/utils/exp_tracking.py:13]: But, F401 will be removed after we clean up all unused imports, isn't it?

Comment (vfdev-5) [reply examples/references/classification/imagenet/code/utils/exp_tracking.py:13]: I was considering that previously but it looks like we will have to add a lot of those `# noqa: F401` inside `__init__.py`, right ?

Comment (ydcjeff) [reply examples/references/classification/imagenet/code/utils/exp_tracking.py:13]: Yes unless we add them in `__all__`

Comment (vfdev-5) [reply examples/references/classification/imagenet/code/utils/exp_tracking.py:13]: Sounds good !

**Review by vfdev-5 — APPROVED** [review]
LGTM!

**Pull request merged**